x-pack/plugins/cross_cluster_replication/index.js

```diff
@@ -38,7 +38,15 @@ export function crossClusterReplication(kibana) {
         enabled: Joi.boolean().default(true),
       }).default();
     },
-
+    isEnabled(config) {
+      return (
+        config.get('xpack.ccr.enabled') &&
+        config.has('xpack.index_management.enabled') &&
+        config.get('xpack.index_management.enabled') &&
+        config.has('xpack.remote_clusters.enabled') &&
+        config.get('xpack.remote_clusters.enabled')
```

Comment:
It looks like `index_management` and `remote_clusters` are each being checked twice. I think we mean this?

```js
      return (
        config.get('xpack.ccr.enabled') &&
        config.has('xpack.index_management.enabled') &&
        config.has('xpack.remote_clusters.enabled')
     );
```

Reply:
"has" checks for the existence of the key and "get" checks for its value.  "get' will blow up if has is false, hence the two checks.

Reply:
not true, it turns out.  Tested get with nonexistent key and it worked fine.